.github/workflows/contributors.yml

```diff
@@ -51,16 +52,17 @@ jobs:
                   CONTRIBUTORS=${{ steps.get_contributors.outputs.contributors }}
                   sed -i 's/<!--CONTRIBUTOR COUNT START-->.*<!--CONTRIBUTOR COUNT END-->/<!--CONTRIBUTOR COUNT START--> '"$CONTRIBUTORS"' <!--CONTRIBUTOR COUNT END-->/g' MAINTAINERS.md
 
-            - name: Commit changes
+            - name: Commit and push changes
               # https://github.com/orgs/community/discussions/26560#discussioncomment-3531273
               # commit changes, but if no changes exist, then exit cleanly
               run: |
                   git config user.name 'github-actions[bot]'
                   git config user.email 'github-actions[bot]@users.noreply.github.com'
                   git add MAINTAINERS.md
                   git commit -m "BOT: Update contributors info in MAINTAINERS.md" || exit 0
+                  git push
```

Comment:
You won't be able to push to main unless you bypass branch protection somehow. That's why I added the `CasperWA/push-protected` action below.